plugins/xdebug/xdebug.cc

```diff
@@ -17,14 +17,18 @@
  */
 
 #include <cstdlib>
+#include <stdio.h>
 #include <cstdio>
 #include <strings.h>
+#include <sstream>
 #include <cstring>
 #include <getopt.h>
```

Comment:
We should prefer the new rather than the obsolete (in C++) C library headers.  cstdio over stdio.h, and cstring over string.h.  Why on earth are we using *both* new and obsolete headers here?